### DISCUSSION TITLE
why is my data not loading?

### DISCUSSION THREAD

**Discussion by jeevan-aj** [version0chiro/Find-Me-Issues#115] [Q&A]
I was trying to fix a issue so i cloned the repo , install packages and run it but the data is not getting loaded .

instead of (cards) data i am seeing "You are seeing this message because github imposes rate limit on requests. Please refresh the page or wait a couple of minutes."

https://github.com/user-attachments/assets/85d58e8f-9485-4b32-a9f2-426388475442

**Comment by MiArianM** [comment]
Hey Jeevan;
It's been a long time since your issue here, and probably the problem has passed. But if you are still working on this project or you encounter an error similar to this again, this is because of a fact that when requests are sent to the back-end server to get data, **if they are not cached**, it will happen which means every series of requests will be fired from the beginning, which will cause you to be unable to re-request for a limited time after refreshing the page 5-8 times while editing it.
Have good Coding sessions out there!
